common/class.tx_dlf_list.php

```diff
@@ -244,35 +244,47 @@ protected function getRecord($element) {
                     $params = array ();
 
                     // Restrict the fields to the required ones
-                    $params['fl'] = 'uid,id,toplevel,thumbnail,page';
+                    $params['fields'] = 'uid,id,toplevel,thumbnail,page';
 
                     foreach ($this->solrConfig as $solr_name) {
 
-                        $params['fl'] .= ','.$solr_name;
+                        $params['fields'] .= ','.$solr_name;
 
                     }
 
-                    // Get document's thumbnail and metadata from Solr index.
-                    $result = $this->solr->service->search('uid:'.tx_dlf_solr::escapeQuery($record['uid']), 0, $this->solr->limit, $params);
-
                     // If it is a fulltext search, enable highlighting and fetch the results
                     if ($this->metadata['fulltextSearch']) {
 
-                        $params = array ();
+                        $params['component'] = array (
+                            'highlighting' => array (
+                                'query' => tx_dlf_solr::escapeQuery($this->metadata['searchString']),
+                                'field' => 'fulltext',
+                                'usefastvectrohighlighter' => true
```

Comment:
- solrconfig.xml:
I thought it would be a good idea to use the new _default version of solrconfig.xml and adjust it. Yes, I did check the old file and anything should be customized as well.
- OK, I just used the stopwords files of the _default configset to have a "complete" one. It is not the problem to remove the unused files if you like?
- Of course, there is sill an authentication option for Solr! I removed it because this configuration is a part of Solr (Jetty) and not of this extension. I would place a hint to the matching pages of the Solr/Jetty documentation?
- I'm not sure about that point. If anyone does't use the composer the extension does't work. Of cource we can write a documentation on that, but maybe it looks daunting at first sight. Independently of that deployment via composer should already work.

Reply:
Thanks, of course it is a typo!